### PR TITLE
Add Pharmacy Union promo corp

### PR DESCRIPTION
**Pull request by nwai90**
**Ref:** https://github.com/bafolts/terraforming-mars/issues/845

Screenshot 2020-06-20 at 6 33 06 PM

### CODE REVIEW

File: src/HTML_data.ts [hunk 1]
```diff
@@ -6793,6 +6793,29 @@ export const HTML_DATA: Map<string, string> =
...
       </div>
 `],
+[CardName.PHARMACY_UNION,`
+      <div class="tag tag1 tag-microbe"></div>
+      <div class="tag tag2 tag-microbe"></div>
+      <div class="corporationLabel">CORPORATION</div>
+      ##RESOURCES##
+      <div class="venus-icon corporation-icon"></div>
+      <div class="contentCorporation">
+        <div class="corporationEffectBox" style="height:150px;margin-top:130px;">
```
Comment (alrusdi) [inline src/HTML_data.ts:6803]: Can you please do not use inline styles?
We have a lot of legacy here, but for new cards it is better to avoid it

Comment (nwai90) [reply src/HTML_data.ts]: Sure, I updated to use classes

File: src/cards/promo/PharmacyUnion.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                );
+                return undefined;
+            } else {
+                const availableOptions: OrOptions = new OrOptions();
+
+                availableOptions.options.push(
+                    new SelectOption('Turn this card face down and gain 3 TR', () => {
+                        this.tags = [];
+                        player.increaseTerraformRatingSteps(3, game);
+                        this.onCardPlayed = () => {};
```
Comment (vincentneko) [inline src/cards/promo/PharmacyUnion.ts:66]: I think we need a boolean to store if the corporation has been disabled.
Using the undo will reset the gain 3 TR option and reinstate the tags, we need to reset the tags and the onCardPlayed based on the boolean value.

Comment (nwai90) [reply src/cards/promo/PharmacyUnion.ts]: I updated; the tags of disabled corporation will also be excluded from `player.getTagCount()`

File: src/HTML_data.ts [hunk 3]
```diff
@@ -6800,19 +6800,19 @@ export const HTML_DATA: Map<string, string> =
       ##RESOURCES##
       <div class="venus-icon corporation-icon"></div>
       <div class="contentCorporation">
-        <div class="corporationEffectBox" style="height:150px;margin-top:130px;">
-          <div class="corporationEffectBoxLabel" style="margin-bottom:5px">EFFECT</div>
+        <div class="corporationEffectBox pharmacy_union-corp-effect-box">
```
Comment (alrusdi) [inline src/HTML_data.ts:6803]: You don't need to apply specific css classes for each block.
Each card has a parent class like card-pharmacy-union.
So it is perfectly correct to just remove inline styles from html and then apply them in css filest like this:

`.card-pharmacy-union .corporationEffectBox {height:150px;margin-top:130px;}`

Do you want me to make an example of it on this card? Or it is crear and you can fix it?

Comment (nwai90) [reply src/HTML_data.ts:6803]: @alrusdi Thanks! I'll give it a shot

Comment (nwai90) [reply src/HTML_data.ts:6803]: Could you make the example? Can't seem to get it working right

Comment (alrusdi) [reply src/HTML_data.ts:6803]: Ok, lets's merge it. Then i will make new markup and you will take it as s reference for upcoming cards

**Pull request merged**